CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2025-01-30
```

Comment:
It looks like the changes from #56 are included in this PR now?
We should update the changelog to also include the "allow arguments from any subcommand" change from this PR and close #56 then

Reply:
Sorry I was testing some stuff to make the merging of these PRs easier, but Github actually had a small outage before I could finish it. These changes have been reverted and done properly now (I had accidentally both rebased and merged #56 into this)